standup/status/management/commands/mergeuser.py

```diff
@@ -77,8 +88,19 @@ def get_status_count(user):
         user_keep.email = user_delete.email
         user_keep.save()
 
-        # Delete.
-        # FIXME(willkg): Hopefully this cascades....
+        # Copy teams from delete -> keep
+        self.stdout.write('Copying teams from delete -> keep')
+        if hasattr(user_delete, 'profile'):
+            for team in user_delete.profile.teams.all():
+                try:
+                    with transaction.atomic():
```

Comment:
Without this line, the transaction model django has gets all fussy. I think it's because I'm lazily figuring things out by seeing if it throws an `IntegrityError` or not, but I'm not positive that's the issue. The error message I get back is this:

> django.db.transaction.TransactionManagementError: An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.